main.py

```diff
@@ -58,10 +60,10 @@ def get_account_server(self, account):
         return plexapi.utils.choose('Select server index', servers, 'name').connect()
 
     def get_server_section(self, server):
-        sections = [ _ for _ in server.library.sections() if _.type in {'movie'} ]
-        if not sections:
-            print('No available sections.')
-            sys.exit()
+         sections = [ _ for _ in server.library.sections() if _.type in {'movie'} ]
```

Comment:
@jwmoss Looks like some extra whitespace got in here and python's bombing on it.